.github/workflows/prod-release.yml

```
@@ -86,10 +86,30 @@ jobs:
         env:
           PROD: true
           OWNER: managerforlustre
-          PROJECT: manager-for-lustre-6.2
+          PROJECT: manager-for-lustre-6.3
           PACKAGE: iml-docker
           SRPM_TASK: iml-docker-srpm
           SPEC: iml-docker.spec
           WORKSPACE: ${{ github.workspace }}
           KEY: ${{ secrets.key }}
           IV: ${{ secrets.iv }}
+
+  deploy_copr:
```

This name should match the rest

```suggestion
  push_python_iml_manager:
```